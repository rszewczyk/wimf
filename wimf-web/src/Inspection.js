// @flow
import React from "react";
import { css } from "glamor";

type InspectionProps = {
  businessName: string,
  violationDescription: string,
  inspectionDate: string
};

const inspectionStyle = css({
  borderBottom: "1px solid gray",
  overflow: "hidden",
  fontSize: "0.875rem",
  "> div h5 ": {
    display: "inline-block",
    marginBottom: "0.5rem",
    marginRight: "1.0rem"
  }
});

export default function Inspection(props: InspectionProps) {
  const { businessName, violationDescription, inspectionDate } = props;

  return (
    <div {...inspectionStyle}>
      <div>
        <h5>{businessName}</h5>
        <span>{inspectionDate}</span>
      </div>
      <p>{violationDescription}</p>
    </div>
  );
}
