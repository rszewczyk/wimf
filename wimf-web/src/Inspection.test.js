import React from "react";
import renderer from "react-test-renderer";
import Inspection from "./Inspection";

it("renders correctly", () => {
  expect(
    renderer
      .create(
        <Inspection
          businessName="foo.bar.org"
          violationDescription="ew"
          inspectionDate="1/2/2016"
        />
      )
      .toJSON()
  ).toMatchSnapshot();
});
