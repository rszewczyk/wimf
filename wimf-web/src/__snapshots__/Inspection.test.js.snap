// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  data-css-1ue8gkg=""
>
  <div>
    <h5>
      foo.bar.org
    </h5>
    <span>
      1/2/2016
    </span>
  </div>
  <p>
    ew
  </p>
</div>
`;
